Packs/jamf/Integrations/jamf_v2/jamf_v2.py

```diff
@@ -0,0 +1,982 @@
+            'Activation Lock Enabled': security_mobile_response.get('activation_lock_enabled'),
+            'Jailbreak Detected': security_mobile_response.get('jailbreak_detected'),
+            'Lost Mode Enabled': security_mobile_response.get('lost_mode_enabled'),
+            'Lost Mode Enforced': security_mobile_response.get('lost_mode_enforced'),
+            'Lost Mode Enable Issued UTC': security_mobile_response.get('lost_mode_enable_issued_utc'),
+            'Lost Mode Message': security_mobile_response.get('lost_mode_message'),
+            'Lost Mode Phone': security_mobile_response.get('lost_mode_phone'),
+            'Lost Mode Footnote': security_mobile_response.get('lost_mode_footnote'),
+            'Phone': security_mobile_response.get('activation_lock_enabled'),
+
+        }
+    elif subset == 'Network':
+        network_mobile_response = mobile_response.get('network')
+
+        readable_output = {
+            'Home Carrier Network': network_mobile_response.get('home_carrier_network'),
+            'Cellular Technology': network_mobile_response.get('cellular_technology'),
+            'Voice_roaming_enabled': network_mobile_response.get('voice_roaming_enabled'),
+            'Imei': network_mobile_response.get('imei'),
+            'Iccid': network_mobile_response.get('iccid'),
+            'Meid': network_mobile_response.get('meid'),
+            'Current Carrier Network': network_mobile_response.get('current_carrier_network'),
+            'Carrier Settings Version': network_mobile_response.get('carrier_settings_version'),
+            'Current Mobile Country Code': network_mobile_response.get('current_mobile_country_code'),
+            'Current Mobile Network Code': network_mobile_response.get('current_mobile_network_code'),
+            'Home Mobile Country Code': network_mobile_response.get('home_mobile_country_code'),
+            'Home Mobile Network Code': network_mobile_response.get('home_mobile_network_code'),
+            'Data Roaming Enabled': network_mobile_response.get('data_roaming_enabled'),
+            'Phone Number': network_mobile_response.get('phone_number')
+        }
+    elif subset == 'ProvisioningProfiles':
+        provisioning_profiles_mobile_response = mobile_response.get('provisioning_profiles')
+
+        readable_output = provisioning_profiles_mobile_response
+    elif subset == 'MobileDeviceGroups':
+        mobile_device_groups_mobile_response = mobile_response.get('mobile_device_groups')
+
+        readable_output = {
+            'Number of groups': len(mobile_device_groups_mobile_response)
+        }
+    return readable_output
+
+
+def get_computers_by_app_readable_output(response):
+    readable_output = []
+    versions_list = response.get('versions')
+    for version in versions_list:
+        computers_len = len(version.get('computers'))
+        readable_output.append({
+            'version': version.get('number'),
+            'Sum of computers': computers_len
```

Comment:
The word `sum` does not fit here IMO.
Perhaps `Total computers` or something like that